### PR TITLE
Update Copy

### PR DESCRIPTION
**Pull request by elliotwhitehead**
## Description
I updated the description of the events on the calendar.

### CODE REVIEW

File: _data/events.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+  line: A different kind of hiring event - Bridging the gap between students and local startups.
   date: 2018-10-23 6:00 PM
   end-date: 2018-10-23 8:00 PM
   location: Idea Forge
   url: http://startups2students.org
 
 - name: Phase
   image-url: assets/img/phase.png
-  line: 16h hardware hackathon
+  line: Our first ever 16 hour hardware hackathon - Come learn and build with hardware!
```
Comment (casassg) [inline _data/events.yml]: 15 hours. We changed this

**Review by casassg — APPROVED** [review]
if you can change the 16 -> 15 hours, that would be cool

File: _data/events.yml (hunk 1 above)
Comment (casassg) [reply _data/events.yml]:
```suggestion
  line: Our first ever 15-hour hardware hackathon - Come to learn and build with hardware!
```

**Comment by elliotwhitehead** [comment]
Good catch @casassg 👀

**Pull request merged**